pkg/common/peertracker/getpid.c

```diff
@@ -0,0 +1,59 @@
+#include "getpid.h"
+#include <iphlpapi.h>
+#include <stdio.h>
+
+int getOwningPIDFromLocalConn(int localPort, int remotePort)

```

Comment:
I wonder if we shouldn't use the following signature:
```
unsigned int getOwningPIDFromLocalConn(int localPort, int remotePort, int *pid)
```

(`unsigned int` is the same definition as `DWORD`, but will allow us to not need to include windows.h from the header file).

This allows us to return the DWORD error code (or NO_ERROR on success), which can then by coerced into a syscall.Errno to give us a nice error message (it implements `error`).

* We could return ERROR_OUT_OF_MEMORY from failures to allocation.
* We could return failures from GetExtendedTcpTable (except ERROR_INSUFFICIENT_BUFFER, which we handle in the loop)
* We could return ERROR_NOT_FOUND if we can't find the connection in the table